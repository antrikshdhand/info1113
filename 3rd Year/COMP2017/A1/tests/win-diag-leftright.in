place D17
place A1
place E16
place C1
place F15
place E1
place G14
place G1
place H13
