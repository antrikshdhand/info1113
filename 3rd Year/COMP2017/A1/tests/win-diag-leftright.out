Black wins!
D17A1E16C1F15E1G14G1H13
Thank you for playing!
